apps/storybook/src/Breadcrumbs.stories.tsx

```diff
@@ -88,6 +102,57 @@ export const Overflow: Story<BreadcrumbsProps> = (args) => {
   );
 };
 
+export const CustomOverflow: Story<BreadcrumbsProps> = (args) => {
+  const items = Array(10)
+    .fill(null)
+    .map((_, index) => (
+      <Button
+        key={index}
+        onClick={() => action(`Clicked on button ${index + 1}`)()}
+      >
+        Item {index}
+      </Button>
+    ));
+
+  return (
+    <div style={{ maxWidth: '50%', border: '1px solid lightpink', padding: 8 }}>
+      <Breadcrumbs
+        overflowButton={(visibleCount) => (
+          <DropdownMenu
+            menuItems={(close) =>
+              Array(items.length - visibleCount)
+                .fill(null)
+                .map((_, _index) => {
+                  const index = visibleCount > 1 ? _index + 1 : _index;
+                  const onClick = () => {
+                    action(`Clicked button ${index} (overflow)`)();
+                    close();
+                  };
+                  return (
+                    <MenuItem
+                      key={index}
+                      onClick={onClick}
+                      icon={<SvgPlaceholder />}
```

Comment:
Icon is not used in orignal breadcrumb so it shouldn't be here too.

Reply:
Fixed